src/source/PeerConnection/SessionDescription.c

```diff
@@ -316,6 +326,64 @@ PCHAR fmtpForPayloadType(UINT64 payloadType, PSessionDescription pSessionDescrip
     return NULL;
 }
 
+/*
+ * Extracts a (hex) value after the provided prefix string. Returns true if
+ * successful.
+ */
+static BOOL readHexValue(const PCHAR input, const PCHAR prefix, PINT32 value) {
```

Comment:
Ok, will pull the const usage out - looks like that's the preferred pattern in this library.

Re static, this makes the function private to this file, which is what I was going for (hide from linker...) - this was to prevent it from appearing in the global symbol space. Would you prefer if I made this function linkable outside this compilation unit (and declare in SessionDescription.h?)

Reply:
We use visibility to expose or not to from the compile unit. All our public APIs are marked as PUBLIC_API